src/main/java/org/dswarm/wikidataimporter/WikidataDswarmImporter.java

```diff
@@ -194,7 +491,23 @@ private PropertyIdValue createOrGetWikidataProperty(final String propertyIdentif
 				final LiteralNode literalNode = (LiteralNode) object;
 				final String value = literalNode.getValue();
 
-				return Optional.ofNullable(Datamodel.makeStringValue(value));
+				final String finalValue;
+
+				if (!value.trim().isEmpty()) {
+
+					// note: we need to trim the values; otherwise, we'll get a 'wikibase-validator-malformed-value' error
+					final String trimmedValue = value.trim();
+
+					// note: we need to cut the values, if they are longer then 400 characters; otherwise, we'll get a 'wikibase-validator-too-long' error
+					finalValue = cutLongValue(trimmedValue);
+				} else {
+
+					// empty values are not possible in Wikidata - insert placeholder for now
+
```

Comment:
yes, at the end it doesn't really matter right now, since this was only a prototypical implementation, i.e., this case was handled "somehow" ;)